GVFS/GVFS.Installer/GVFS.Installer.Mac.csproj

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
In Visual Studio for Mac, this project is showing files that are for Windows only (packages.config and Setup.iss).

I think the real issue is that you should create two separate folders for the Mac and Windows installers here, rather than have two csproj files in the same folder. The projects that we have that put two csproj's into the same folder do so because those csproj's build the same source files, just targeted to different runtimes, e.g. GVFS.Mac.csproj and GVFS.Windows.csproj. However, in cases where they build different entirely different source files, we create separate folders for them as well, e.g. GVFS.Platform.Mac and GVFS.Platform.Windows. The installer is much more like the latter example.

Reply:
Created separate directories for Mac & Windows installer projects.